src/containers/GalaxiesSelectorContainer.jsx

```diff
@@ -33,7 +57,7 @@ class GalaxiesSelectorContainer extends React.PureComponent {
       this.setState(prevState => ({
         ...prevState,
         data: objects,
-        imagePath: `/images/galaxies/hsc/${name}.png`,
+        imagePath: `/images/galaxies/hsc/${name}.jpg`,
```

Comment:
I'm understanding that from this, there will always be a compressed jpg file format and not a png?

Reply:
sure.  In this case the pngs were just way too big and were slowing page loads down so I elected to update the files used.

Reply:
👏 Good decision, then!!!!